src/v1_0/rw/ctrl.cpp

```diff
@@ -2,13 +2,14 @@
 
 namespace abb :: rws :: v1_0 :: rw :: ctrl
 {
-    void restartController(RWSClient& client)
+    void restartController(RWSClient& client, RestartMode const& restartMode)
     {
         std::stringstream uri;
         uri << Services::CTRL;
 
-        std::string content = "restart-mode=restart";
+        std::stringstream content;
+        content << "restart-mode=" << restartMode;
 
-        client.httpPost(uri.str(), content);
+        client.httpPost(uri.str(), content.str(), std::set<Poco::Net::HTTPResponse::HTTPStatus> {Poco::Net::HTTPResponse::HTTP_NO_CONTENT, Poco::Net::HTTPResponse::HTTP_OK});
```

Comment:
Why two codes are accepted?

Reply:
When I tested it, my controller returned HTTP_OK, but according to RWS1 documentation it says both can be returned, so I'm not sure whether on a different RW6 version a different code cannot be returned. Since both are of the success group I kept both

Reply:
Makes sense. Could you point me to the place where it says both codes can be returned?